client/src/components/Layout/Footer.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import styled from 'styled-components';
 
-const Footer = props => {
+const Footer = () => {
```

Comment:
Should I give you props for this :joebacus:

Reply:
🔥🔥🔥